public/javascripts/index.js

```diff
@@ -6,52 +6,59 @@ function scrollToElement (element) {
 }
 
 function renderError (el, message) {
-  console.log('Appending: ' + message)
-  el.text(' - ' + message)
+  console.log(message)
```

Comment:
ah, right. "dvcoders-backend" made me think that all of the files are entirely server-side code.
yeah, let's remove that then, don't think it's useful in anyway?